data-prepper-plugins/opensearch/src/test/java/com/amazon/dataprepper/plugins/sink/opensearch/OpenSearchSinkIT.java

```diff
@@ -96,7 +96,7 @@ public void testInstantiateSinkRawSpanDefault() throws IOException {
     assertFalse((boolean)mappings.get("date_detection"));
     sink.shutdown();
 
-    if (isODFE()) {
+    if (isOSBundle()) {
```

Comment:
What is the purpose of `isODFE/isOSBundle`? From what I can tell it enables certain assertions and configures if a client should use http or https. A few follow-up questions:

1. Would the OpenSearchSink ever be tested on just OpenSearch (without bundled plugins)?
1. Why does bundle/no bundle change the test behavior?
1. Why does http/s depend on the bundle flag?

Reply:
> 1. Would the OpenSearchSink ever be tested on just OpenSearch (without bundled plugins)?
Yes. `./gradlew build` will run on OpenSearch core only.
> 2. Why does bundle/no bundle change the test behavior?
AFAICT, Bundle introduces
(1) security configuration (e.g. username/password) needed for security plugin
(2) Index management policy needed for ISM plugin 
> 3. Why does http/s depend on the bundle flag?
By default, bundle has security plugin enabled which requires https header.

Reply:
Thanks for the info. Would suggest somehow changing the variable/method names (or even adding a comment) to better explain what the "isBundle" method is doing. Not blocking